.gitattributes

```diff
@@ -0,0 +1,9 @@
+/CHANGELOG.md merge=union
+*.pdf binary
+*.doc binary
+*.docx binary
+*.ppt binary
+*.pptx binary
+*.xls binary
+*.xlsx binary
+*.xps binary
```

Comment:
I would probably just delete lines 2-9. They have no effect on the Tensile repository, as you don't have any of those file types.

That said, it's no big deal either way.

Reply:
> I would probably just delete lines 2-9. They have no effect on the Tensile repository, as you don't have any of those file types.

Who knows where future lies?  Thanks for the comments though.